src/main/resources/com/veertu/plugin/anka/AnkaCloudSlaveTemplate/config.jelly

```diff
@@ -1,44 +1,32 @@
     <j:jelly xmlns:j="jelly:core"  xmlns:f="/lib/form" xmlns:c="/lib/credentials">
 
-            <f:entry title="cloudName" field="cloudName">
-                <f:textbox value="${cloud.getCloudName()}" disabled="true"  />
-            </f:entry>
-
-        <f:entry title="${%Template}" field="masterVmId" description="The snapshot to install jenkins slave on it.
-                                                                        java is assumed to be pre-installed">
-            <f:select />
-        </f:entry>
-        <f:entry title="${%Template Version Tag}" field="tag" description="The tag to use for this vm template">
-            <f:select />
+        <f:entry title="Cloud Name" field="cloudName" description="The Anka Build Cloud this Node Template/Label is attached to">
+            <f:textbox value="${cloud.getCloudName()}" disabled="true" />
         </f:entry>
 
-
-
-        <f:entry title="${%Description}" field="templateDescription">
-            <f:textbox />
+        <f:entry title="${%Target Label}" field="label" description="The label that jobs will target in order to start and utilize the started Jenkins Node/Agent">
+            <f:textbox clazz="required" />
         </f:entry>
 
-        <f:entry title="${%# of Executors}" field="numberOfExecutors" description="This is the number of builds which can run on the same slave">
-            <f:textbox clazz="required number" default="1" />
+        <f:entry title="${%Node/Agent Name}" field="nameTemplate" description="A string to include in the name of the started Jenkins Node/Agent">
```

Comment:
Change title to "Jenkins Node/Agent Name", to be clear that users do not confuse this with Anka Cloud nodes